templates/articles/procurement_approach.html

```diff
@@ -0,0 +1,31 @@
+<html>
```

Comment:
There shouldn't be an `html` element here, only the elements inside of it. I don't think this will render properly.